src/main/java/com/bruno/config/HibernationConfiguration.java

```diff
@@ -0,0 +1,62 @@
+package com.bruno.config;
+
+import java.util.Properties;
+
+import javax.sql.DataSource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.core.env.Environment;
+import org.springframework.jdbc.datasource.DriverManagerDataSource;
+import org.springframework.orm.hibernate5.HibernateTransactionManager;
+import org.springframework.orm.hibernate5.LocalSessionFactoryBean;
+import org.springframework.transaction.PlatformTransactionManager;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+@Configuration
+@EnableTransactionManagement
+@ComponentScan({"com.bruno"})
+@PropertySource(value = { "classpath:application.properties" })
+public class HibernationConfiguration {
+
+	@Autowired
+    private Environment environment;
```

Comment:
Formatting

Reply:
That's weird, those two lines look inline on my IDE but I just changed it on notepad, thanks!